manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -4,7 +4,11 @@ metadata:
   name: ztunnel
   namespace: {{ .Release.Namespace }}
   labels:
-    {{- .Values.labels | toYaml | nindent 4}}
+    istio.io/component: ztunnel
+    operator.istio.io/component: Ztunnel
+    k8s-app: ztunnel
+    app: ztunnel
```

Comment:
```suggestion
    app.kubernetes.io/name: ztunnel
    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
    app.kubernetes.io/instance: {{ .Release.Name }}
```

Reply:
could we also have `app.kubernetes.io/part-of: istio`?

Reply:
(there's maybe a case for including

`app.kubernetes.io/part-of: istio`

 but trying to stick to the minimal recs here)

Reply:
@ilrudie beat me to it. I think so, even though it wouldn't be much use to people until we add it to the other charts (which we will have to do at some point)

Reply:
I'd wager some value might be if someone chooses to install zt not in istio-system then it helps someone not that familiar with ambient figure out what it is